samples/custom-bootstrap/README.md

```diff
@@ -1,18 +1,22 @@
 # Custom Envoy Bootstrap Configuration
 
-This sample creates a simple helloworld service that bootstraps the Envoy proxy with a custom configuration file.
+This sample creates a simple helloworld service that bootstraps the Envoy proxy. There are 2 ways to implement this:
+1. With a custom configuration file. 
+2. With envoy filter
```

Comment:
As this is an md file, you label each as 1., and when viewed, they show correctly:
```suggestion
1. With envoy filter
```